src/env.cc

```diff
@@ -917,6 +917,21 @@ void AsyncHooks::grow_async_ids_stack() {
 uv_key_t Environment::thread_local_env = {};
 
 void Environment::Exit(int exit_code) {
+  if (options()->trace_exit) {
+    HandleScope handle_scope(isolate());
+
+    if (is_main_thread()) {
+      fprintf(stderr, "(node:%d) ", uv_os_getpid());
+    } else {
+      fprintf(stderr, "(node:%d, thread:%llu) ", uv_os_getpid(), thread_id());
+    }
+
+    fprintf(
+        stderr, "WARNING: Exited the environment with code %d\n", exit_code);
```

Comment:
Why do we mark this as a warning and pipe to `stderr`?

Reply:
Other traces were printed as warnings and piped to `stderr`. Do you have any suggestions on the format of the traces?

Reply:
Feel free to resolve this by yourself, I'm okay if this is a default behavior :)